src/components/login/login.js

```diff
@@ -1,63 +1,106 @@
 import React from 'react';
+import { Redirect } from 'react-router-dom';
 import { Form, Input, Button } from 'antd';
 import { UserOutlined, LockOutlined } from '@ant-design/icons';
+import  HTTPClient  from '../../../src/HTTPClient';
 
 import './login.css';
 
-export const LoginForm = () => {
-  const onFinish = values => {
+export class LoginForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state={
+      username:'',
+      password:'',
+      login:false
+    }
+  }
+  onFinish(values) {
     console.log('Received values of form: ', values);
   };
 
-  return (
-    <Form
-      name="normal_login"
-      className="form-box"
-      initialValues={{
-        remember: true,
-      }}
-      onFinish={onFinish}
-    >
-      <Form.Item className="input-field"
-        name="username"
-        rules={[
-          {
-            required: true,
-            message: 'Please input your Username!',
-          },
-        ]}
-      >
-        <Input prefix={<UserOutlined className="site-form-item-icon" />} placeholder="Username" />
-      </Form.Item>
-      <Form.Item
-        name="password"
-        rules={[
-          {
-            required: true,
-            message: 'Please input your Password!',
-          },
-        ]}
-      >
-        <Input
-          prefix={<LockOutlined className="site-form-item-icon" />}
-          type="password"
-          placeholder="Password"
-        />
-      </Form.Item>
+  handleClick(event) {
+    var payload = {
+      "email": this.state.username,
+      "password": this.state.password,
+    }
+    const client = new HTTPClient(process.env.REACT_APP_API_URL,{});
 
-      <Form.Item>
-        <Form.Item>
-          <a className="login-form-forgot" href="">
-            Forgot password?
-          </a>
+    client.post(process.env.REACT_APP_API_URL, payload).then(
+      (response)=>{
+          if (!response.success) {
+            window.alert('Login Unsucessful!' + response.msg);
```

Comment:
@tayoogunbiyi I tried using addtoast and ran into some hook related errors, have use window.alert instead for the time being. will update in seperate commit once fixed.

Reply:
great. could you focus on the create house hookup while I finish this up? you can leave out the toasts it's a bit tricky to work out